dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/discovery/SearchFacetFilter.java

```diff
@@ -452,13 +469,14 @@ private String getNextPageURL(SearchFilterParam browseParams, Request request) t
 
     private String getPreviousPageURL(SearchFilterParam browseParams, Request request) throws UnsupportedEncodingException, UIException {
         //If our offset should be 0 then we shouldn't be able to view a previous page url
-        if (0 == queryArgs.getFacetOffset() && Util.getIntParameter(request, "offset") == -1)
+        boolean currentOffsetSmallerOrEqualTO0 = currentOffset <= 0;
```

Comment:
It would be better to use a variable name that's tied to the meaning -- something like `canGoBack` or `linkToPreviousPage`. At the very least, avoid having the upper-case letter O right next to the digit 0.